### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by freddykrunn**

### CODE REVIEW

File: generators/widget/templates/widget.ts [hunk 1]
```diff
@@ -21,15 +21,28 @@ let outputs: Map<string, Widget.WidgetOutput> = new Map<string, Widget.WidgetOut
...
+ * 
+ * ### Widget Outputs
+ * (Provide a detailed list of the widget outputs here as the example of Widget Inputs above)
+ * 
+ * ### Inputs
+ * (Provide a detailed list of the component inputs here as the example of Widget Inputs above)
  * 
- * ## Example
+ * ### Outputs
+ * (Provide a detailed list of the component outputs here as the example of Widget Inputs above)
```
Comment (jpsfs) [inline generators/widget/templates/widget.ts]: Copy/paste issue. Please change to outputs.

File: generators/widget/templates/widget.ts [hunk 2]
```diff
@@ -21,15 +21,28 @@ let outputs: Map<string, Widget.WidgetOutput> = new Map<string, Widget.WidgetOut
...
+ * 
+ * ### Widget Inputs
+ * * `string` : **inputA** - Widget Input A description
+ * * `number` : **inputB** - Widget Input B description
+ * * `Object[]` : **inputC** - Widget Input C description
+ * * `boolean` : **dynamicInputD** _(dynamic)_ - Widget Dynamic Input D description
+ * (Provide a detailed list of the widget inputs here as the example above)
+ * 
+ * ### Widget Outputs
+ * (Provide a detailed list of the widget outputs here as the example of Widget Inputs above)
```
Comment (jpsfs) [inline generators/widget/templates/widget.ts]: Outputs

**Review by jpsfs — CHANGES_REQUESTED** [review]
Seems good to me!

**Comment by pesimeao** [comment]
@freddykrunn 
Maybe we can add an explanation/example about what is a good description.
I think that the outputs and inputs shouldn't reference other sections. It's more clear to have it "there" IMO.

**Review by jpsfs — APPROVED** [review]

**Pull request merged**